.github/workflows/build-test-windows.yml

```diff
@@ -2,11 +2,22 @@
 name: build-and-test-windows
 on:
   push:
-    branches: [ main, master ]
+    branches:
+      - master
+
+    paths-ignore:
+      - '.github/**'
+      - '!.github/workflows/**'
+      - '**.md'
```

Comment:
Order matters in this configuration: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-and-excluding-paths

So shouldn't `**.md` go first, so that it future-proofs us if we ever put a markdown file in `./.github/workflows/`?

Reply:
Make sense to me. Fixed it in the newest commit though.